frontend/src/components/MapView/DateSelector/utils.ts

```diff
@@ -22,19 +22,21 @@ export function findClosestDate(
   date: number,
   availableDates: ReturnType<Date['getTime']>[],
 ) {
-  const dateToCheck = moment(date);
-
   // TODO - better handle empty arrays.
   if (availableDates.length === 0) {
-    return dateToCheck;
+    return date;
```

Comment:
The function docstring is not accurate anymore. Should this function return a date object maybe? Not sure how it's used atm

Reply:
updated the docstring to number, timestamp in milliseconds